src/resources/Fields/Fields.ts

```diff
@@ -42,4 +42,8 @@ export default class Field extends Resource {
     list(params?: ListFieldsParams) {
         return this.api.get<PageModel<FieldModel>>(this.buildPath(`${Indexes.baseUrl}/page/fields`, params));
```

Comment:
Is this method deprecated?

Reply:
Deprecated, but still supported yes.

Reply:
Maybe we could add a `@deprecated` tag on top of the method to prevent future confusion.